skyportal/handlers/api/source.py

```diff
@@ -939,6 +940,32 @@ def post_source(data, user_id, session, refresh_source=True):
                 )
                 break
 
+        # if there is releases with automatically_publish and one of the source groups,
+        # a public page is published
+        releases = session.scalars(
+            PublicRelease.select(session.user_or_token).where(
+                PublicRelease.groups.any(id=group.id),
+                PublicRelease.automatically_publish,
+            )
+        ).all()
+        if releases is not None and len(releases) > 0:
```

Comment:
i don't think it could be None here right? I suppose that the ORM would always return lists. Worth checking to avoid unnecessary if statements

Reply:
you could also imagine to do `if len(releases) == 0 -> return or whatever you need to do because returning` rather than `if len(releases) > 0 -> a ton of logic that needs to be indentated`. I like to follow this "never indent" principle where possible. Just an advice! Not a requirement at all :)

Reply:
the less indentation, the easier to read